pkg/compiler/callbacks.go

```diff
@@ -37,8 +37,6 @@ type CbAssertT func(ctx context.Context) error
 type RuntimeI interface {
 	NewCbMatch(params MatchParamsT) CbMatchT
 	NewCbAssert(params AssertParamsT) CbAssertT
-	LoadAssertObject(ctx context.Context, obj *ObjT) error
```

Comment:
Is there a corresponding PR in prequel-machine for https://github.com/prequel-dev/prequel-machine/blob/main/tests/compiler_test.go#L330?